src/Sulu/Bundle/AdminBundle/Resources/js/containers/DeleteDependantResourcesDialog/DeleteDependantResourcesDialog.js

```diff
@@ -0,0 +1,261 @@
+// @flow
+import React from 'react';
+import {observer} from 'mobx-react';
+import {toJS, action, computed, observable} from 'mobx';
+import equals from 'fast-deep-equal';
+import Dialog from '../../components/Dialog';
+import {type SnackbarType} from '../../components/Snackbar';
+import ProgressBar from '../../components/ProgressBar';
+import ResourceRequester from '../../services/ResourceRequester';
+import RequestPromise from '../../services/Requester/RequestPromise';
+import {translate} from '../../utils';
+import styles from './deleteDependantResourcesDialogStyles.scss';
+import type {Resource, DependantResourcesData, DependantResourceBatches} from '../../types';
+
+type Props = {
+    dependantResourcesData: DependantResourcesData,
+    onCancel?: () => void,
+    onClose?: () => void,
```

Comment:
i think the `onCancel` is only needed in the `List` container because of the `this.resolveDelete` handling, right? would be nice to ve able to remove both of it, the `onCancel` and the `this.resolveDelete` handli 😇